File: AR7ExportToolbar/ToolStripExport/Exporters/HtmlSectionReportExporter.cs
using System;
using System.Collections.Generic;
using System.Collections.Specialized;
using System.Linq;
using System.Text;
using GrapeCity.ActiveReports.Document;

namespace AR7ExportToolbar.ToolStripExport.Exporters
{
	public class HtmlSectionReportExporter : SectionReportExporter
	{
		private const string TypeName = "GrapeCity.ActiveReports.Export.Html.Section.HtmlExport";
		private const string AssemblyName = "GrapeCity.ActiveReports.Export.Html.v7";

		public HtmlSectionReportExporter()
			: base(TypeName, AssemblyName)
		{
		}

		public override void Export(SectionDocument document, string filename, NameValueCollection settings)
		{
			throw new NotImplementedException();
		}

		public override string DefaultExtension
		{
			get { return "html"; }
		}

		public override string FileDialogFilter
		{
			get { return "HTML Files (*.html)|*.html|All Files (*.*)|*.*"; }
		}
	}
}

File: AR7ExportToolbarVB/ToolStripExport/Exporters/HtmlSectionReportExporter.vb
﻿Imports GrapeCity.ActiveReports.Document
Imports System.Collections.Specialized

Public Class HtmlSectionReportExporter
	Inherits SectionReportExporter
	Private Const TypeName As String = "GrapeCity.ActiveReports.Export.Html.Section.HtmlExport"
	Private Const AssemblyName As String = "GrapeCity.ActiveReports.Export.Html.v7"

	Public Sub New()
		MyBase.New(TypeName, AssemblyName)
	End Sub

	Public Overrides Sub Export(document As SectionDocument, filename As String, settings As NameValueCollection)
		Throw New NotImplementedException()
	End Sub

	Public Overrides ReadOnly Property DefaultExtension() As String
		Get
			Return "html"
		End Get
	End Property

	Public Overrides ReadOnly Property FileDialogFilter() As String
		Get
			Return "HTML Files (*.html)|*.html|All Files (*.*)|*.*"
		End Get
	End Property
End Class
